SingularityBase/src/main/java/com/hubspot/singularity/SingularityRequest.java

```diff
@@ -206,7 +206,7 @@ public boolean isAlwaysRunning() {
 
   @JsonIgnore
   public boolean isOneOff() {
-    return requestType == RequestType.ON_DEMAND;
+    return requestType == RequestType.ON_DEMAND || requestType == RequestType.RUN_ONCE;
```

Comment:
@wsorenson do you agree with this approach, or would you rather I update the check in [SingularityStartup](https://github.com/HubSpot/Singularity/blob/master/SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityStartup.java#L138)?